### DISCUSSION TITLE
Updated GL. Nice!

### DISCUSSION THREAD

**Discussion by brot-senpai** [GLVis/glvis#131] [General]
I see the gl codes has been updated. Nice! I checked it out a few weeks ago and noticed lots of old stuff and got lost trying to find documentation for it because many had been deprecated.  I’ll dig through it again. Thanks!!!

**Comment by tzanio** [comment]
Hi @brot-senpai,

Did you get a chance to give the new version a try? Any feedback?

Cheers,
Tzanio
